src/main/java/ch/njol/skript/expressions/ExprWorldBorderSize.java

```diff
@@ -0,0 +1,88 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.WorldBorder;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Size of World Border")
+@Description("The size of a world border.")
+@Examples("set border radius of {_worldborder} to 10")
+@Since("INSERT VERSION")
+public class ExprWorldBorderSize extends SimplePropertyExpression<WorldBorder, Double> {
+	static {
+		register(ExprWorldBorderSize.class, Double.class, "[[world[ ]]border] (radius|:diameter)", "worldborders");
+	}
```

Comment:
Due to how the existing changers work there is no difference between setting radius or diameter this should be mentioned somewhere within description annotation or changes made accordingly

Reply:
wdym there is no difference?

Reply:
sorry nvm I didn't check good enough the place u set input manages it